src/patternfly/components/InputGroup/examples/input-group-example.hbs

```diff
@@ -85,14 +85,16 @@
 {{#> input-group}}
   {{#> form-control controlType="input" input="true" form-control--attribute='type="search" id="textInput11" name="textInput11" aria-label="search input example"'}}
   {{/form-control}}
-  {{#> button button--modifier="pf-m-tertiary" button--HasSearchIcon="true" button--attribute='aria-label="search button for search input"'}}
+  {{#> button button--modifier="pf-m-tertiary" button--attribute='aria-label="search button for search input"'}}
+    {{> button-icon button-icon--type="search"}}
   {{/button}}
 {{/input-group}}
 <br>
 <br>
 {{#> input-group}}
   {{#> form-control controlType="input" input="true" form-control--attribute='type="text" id="textInput10" name="textInput10" aria-label="input example with popover"'}}
   {{/form-control}}
-  {{#> button button--modifier="pf-m-tertiary" button--HasQuestionCircleIcon="true" button--attribute='aria-label="popover for input"'}}
+  {{#> button button--modifier="pf-m-tertiary"  button--attribute='aria-label="popover for input"'}}
```

Comment:
can you remove the extra space before `button--attribute`?

Reply:
spacing adjusted.